entity-api-spec.yaml

```diff
@@ -2066,6 +2360,11 @@ paths:
                 type: array
                 items:
                   type: object
+              examples:
+                samplearray:
+                  value: 
+                    - collection: 'Sample object 1'
+                    - collection: 'Sample object 2'
```

Comment:
The result of `GET /entities/{id}/samples` is a json array of organ objects, and not related to any `collection` key.